CAM Platform/api-providers.md

```diff
@@ -220,16 +215,14 @@ ElasticBox-Release: 4.0
 }
 ```
 
-**Azure request parameter**
-
+#### Azure request parameter
```

Comment:
Please verify, I think this is missing the `subscription_id`, `client_id`, `secret` and `tenant`.

Reply:
I think this should be approached in the same way as AWS:
- `subscription_id`, `client_id`, `tenant` and `secret` are required for `account_type` = `password`, which means:
  - any ARM provider in an Appliance,
  - normal (not reselling) providers in SaaS,
- `target_email` and `domain` are required for *Migrate my account* (only in SaaS, `account_type` = `existing`)
- nothing else is needed for new accounts (`account_type` = `new`)